src/mtcp/mtcp_restart.c

```diff
@@ -457,9 +459,9 @@ restart_fast_path()
   // rinfo.restorememoryareas_fptr(&rinfo);
   // Intel icc-13.1.3 output uses register rbp here.  It's no longer available.
   asm volatile(
-   // 104 = offsetof(RestoreInfo, rinfo.restorememoryareas_fptr)
+   // 112 = offsetof(RestoreInfo, rinfo.restorememoryareas_fptr)
```

Comment:
With the recent commit by @karya0 , you may find that this comment and assembly code has disappeared.  If so, that's fine -- not a problem.